src/tools/file-format-verification/index.jsx

```diff
@@ -40,37 +39,33 @@ class FFVT extends React.Component {
   }
 
   render() {
-    const selector = <FilingPeriodSelectorContainer />
     const { maintenanceMode, ffvtAnnouncement, filingAnnouncement } = this.props.config
     const downtimeMessage = ffvtAnnouncement || (filingAnnouncement && filingAnnouncement.message)
     return (
       <Provider store={store}>
         <AppContainer>
-          <div id="main-content" className="grid FFVT">
+          <div id='main-content' className='grid FFVT'>
```

Comment:
just a quick comment, I think the standards are still to have double quotes for html attributes even within jsx.